cmake/DaemonGame.cmake

```diff
@@ -200,4 +250,6 @@ function(GAMEMODULE)
             endforeach()
         endif()
     endif()
+
+    set(NACL_TARGETS "${NACL_TARGETS}" PARENT_SCOPE)
```

Comment:
Now NACL_TARGETS is only used within this function, right? So it is not needed in another scope.